src/components/editor/diacritics/VocabChoice.jsx

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import specialcharacters from '../../../../static/specialcharacters.json'
```

Comment:
Can this just be `static/specialCharacters`?

Reply:
I'm not able to get `static/specialcharacters.json` to compile.